config/routes.rb

```diff
@@ -1,7 +1,5 @@
 Rails.application.routes.draw do
-  namespace :admin do
-    resources :teams, only: %i[index new create]
-  end
+  resources :teams, path: 'admin/teams', only: %i[index new create]
```

Comment:
Did you move teams out of the admin namespace?

Reply:
Yes he did, nice one Jamie